indico/modules/events/paper_reviewing/controllers.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import mimetypes
+
+from flask import flash, redirect, request, session
+
+from indico.core.db import db
+from indico.modules.events.paper_reviewing.models.papers import PaperFile
+from indico.modules.events.contributions.controllers.management import RHManageContributionBase
+from indico.modules.events.contributions.controllers.display import RHContributionDisplayBase
+from indico.util.i18n import ngettext, _
+from indico.util.fs import secure_filename
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+from indico.web.util import jsonify_data
+
+# TODO: Access control
+
+def _render_paper_file_list(contrib):
+    tpl = get_template_module('events/contributions/display/_paper_reviewing.html')
+    return tpl.render_paper_files(contrib.paper_files.filter_by(revision_id=None))
```

Comment:
I'd call `.all()` here to avoid passing a query object (that'd probably send the query every time it's iterated over) to the macro